openmc/mgxs/mgxs.py

```diff
@@ -5138,14 +4696,14 @@ class Chi(MGXS):
 
     """
 
-    def __init__(self, domain=None, domain_type=None,
-                 groups=None, by_nuclide=False, name='', num_polar=1,
-                 num_azimuthal=1):
+    def __init__(self, domain=None, domain_type=None, groups=None,
+                 by_nuclide=False, name='', num_polar=1, num_azimuthal=1):
         super(Chi, self).__init__(domain, domain_type, groups, by_nuclide,
                                   name, num_polar, num_azimuthal)
         self._rxn_type = 'chi'
         self._estimator = 'analog'
         self._valid_estimators = ['analog']
+        self.nu = True
```

Comment:
It defaults to True for chi because the we use `nu-fission` weighting of the spectrum to capture the variation in the multiplicities effect on the Eout distributions.